ceph-nvmeof.conf

```diff
@@ -36,5 +36,5 @@ timeout = 60.0
 log_level = ERROR
 # conn_retries = 10
 tgt_cmd_extra_args =
-transports = tcp
+# transports = tcp
 # transport_tcp_options = {"max_queue_depth" : 16, "max_io_size" : 4194304, "io_unit_size" : 1048576, "zcopy" : false}
```

Comment:
Actually if we want the commented params to always show the defaults, then `transport_tcp_options` is not valid here. This commented string is not the default but just an example.

May be should change it to show the default values? The most seems to be found in this file [1]. I suppose only "zcopy" is not initialized there.

[1] https://github.com/ceph/spdk/blob/10edc60aa8b5f1b04d6496fea976dec75e276a95/lib/nvmf/tcp.c#L3377

Reply:
> Actually if we want the commented params to always show the defaults, then `transport_tcp_options` is not valid here.

I don't think it's really necessary, the point is just to make options that are required stand out so that it's clear what a minimal config file is.  Doing this for `transport_tcp_options` would be particularly challenging since the defaults come from another project and could easily go stale (since the gateway isn't tied to the bundled copy of SPDK).